src/test/scala/org/broadinstitute/dsde/firecloud/MockServers.scala

```diff
@@ -136,6 +184,47 @@ object MockServers {
           )
           .withStatusCode(OK.intValue)
       )
+
+    // entity-level responses
+
+    MockServers.workspaceServer
+      .when(
+        request()
+          .withMethod("POST")
+          .withPath(s"/workspaces/${mockSampleValid.wsNamespace}/${mockSampleValid.wsName}/entities")
+          .withBody(mockSampleValid.toJson.compactPrint)
+          .withCookies(cookie)
+      ).respond(
+        response()
+          .withHeaders(header)
+          .withStatusCode(Created.intValue)
+      )
+
+    MockServers.workspaceServer
+      .when(
+        request()
+          .withMethod("POST")
+          .withPath(s"/workspaces/${mockPairValid.wsNamespace}/${mockPairValid.wsName}/entities")
+          .withBody(mockPairValid.toJson.compactPrint)
+          .withCookies(cookie)
+      ).respond(
+        response()
+          .withHeaders(header)
+          .withStatusCode(Created.intValue)
+      )
+
+    MockServers.workspaceServer
+      .when(
+        request()
+          .withMethod("POST")
+          .withPath(s"/workspaces/${mockSampleConflict.wsNamespace}/${mockSampleConflict.wsName}/entities")
+          .withBody(mockSampleConflict.toJson.compactPrint)
+          .withCookies(cookie)
+      ).respond(
+        response()
+          .withHeaders(header)
+          .withStatusCode(Conflict.intValue)
+      )
```

Comment:
I don't mean any sort of live server, I'm wondering if we can mock by instead running a local service using an in-memory database during the life of the unit test - it may turn out this isn't very feasible to do, but I just worry about mocking out the functionality the same as Rawls provides and running unit tests that pass despite changes to Rawls that would break integration tests